langchain/embeddings/openai.py

```diff
@@ -106,7 +106,7 @@ class OpenAIEmbeddings(BaseModel, Embeddings):
     client: Any  #: :meta private:
     model: str = "text-embedding-ada-002"
     deployment: str = model  # to support Azure OpenAI Service custom deployment names
-    openai_api_version: str = "2022-12-01"
+    openai_api_version: str = ""
```

Comment:
Why make default value `""` you can have a default value at https://github.com/hwchase17/langchain/pull/4234/files#diff-d7a6d6752d04755e0881f71a73e707932889769c486afac9ede77b92ea125cbbR147

Reply:
@pranjaldoshi96 I made the default value `""` according to other azure model implementations: https://github.com/zijie0/langchain/blob/96f4467b3c26eceae28cedd6a5180adbcda6cd9e/langchain/chat_models/azure_openai.py#L45

By making the default `""`, we are forcing user to define this variable either in env or in named parameter. 

You remind me that this parameter should be optional with default OpenAI embeddings. So I also added `openai_api_type` check in validation.